packages/react-core/src/components/Chip/__tests__/ChipGroup.test.tsx

```diff
@@ -6,65 +6,63 @@ import userEvent from '@testing-library/user-event';
 import { ChipGroup } from '../index';
 import { Chip } from '../../Chip';
 
-describe('ChipGroup', () => {
-  test('chip group default', () => {
-    const { asFragment } = render(
-      <ChipGroup>
-        <Chip>1.1</Chip>
-      </ChipGroup>
-    );
+test('chip group default', () => {
+  render(
+    <ChipGroup>
+      <Chip>1.1</Chip>
+    </ChipGroup>
+  );
 
-    expect(asFragment()).toMatchSnapshot();
-  });
+  expect(screen.getByRole('group')).toHaveClass('pf-v5-c-chip-group');
```

Comment:
Can you match what Adam is doing in his [PR](https://github.com/patternfly/patternfly-react/pull/9266/files) to not hav hard code class names in our test.  This will make upgrading to later version of styles easier in the future.